css/card.css

```diff
@@ -389,6 +389,50 @@ section#gacha-controls fieldset:nth-child(2) > input:first-child {
   display: none;
 }
 
+@media screen and (min-width: 1600px) {
+  .details-dialog-card {
```

Comment:
there are a lot of details-dialog CSS rules throughout this file and I wonder if they can be consolidated? Don't spend too much time though since we are short on time